index.html

```diff
@@ -22,7 +22,7 @@ <h2>The Void (Linux) distribution</h2>
 		</p>
 		<p>It is available for a variety of platforms. Software packages can be built natively or cross compiled through the <a href="https://github.com/void-linux/void-packages">XBPS source packages collection</a>.
 		</p>
-		<p>Follow us on <a href="https://twitter.com/voidlinux" title="Void on Twitter">Twitter</a>, visit the <a href="ircs://irc.freenode.net/#voidlinux">#voidlinux</a> IRC channel on irc.freenode.net, and join the <a href="https://www.reddit.com/r/voidlinux/">Void Linux subreddit</a>.
+		<p>Follow us on <a href="https://twitter.com/voidlinux" title="Void on Twitter">Twitter</a>, visit the <a href="ircs://irc.libera.chat/#voidlinux">#voidlinux</a> IRC channel on irc.libera.chat, and join the <a href="https://www.reddit.com/r/voidlinux/">Void Linux subreddit</a>.
```

Comment:
It would be nice to link the libera.chat website:

`... IRC channel on irc.libera.chat`
->
`... IRC channel on <a href="https://libera.chat">libera.chat</a>`

Reply:
But then it will be a link within a link...

Reply:
Oh no, sorry

Reply:
Done, thanks!